pkgs/development/interpreters/ruby/default.nix

```
@@ -256,6 +256,10 @@ let
     ) args; in self;
 
 in {
+  inherit rubyVersion;
+
+  rubyGenericBuilder = generic;
```

```suggestion
  mkRubyVersion = rubyVersion;

  mkRubyGenericBuilder = generic;
```
A lot of constructors in nixpkgs start with `mk`, so let's follow that convention.

---

I've renamed them to `mkRubyVersion` and `mkRuby` (inspired by https://github.com/NixOS/nixpkgs/pull/55636). Let me know if you prefer `mkRubyGenericBuilder`.

---

That's fine, thanks